### PR TITLE
feat: view home page

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**
resolves https://github.com/holdex/marketing/issues/90

Test: https://thirdweb-svelte-git-feat-home-clearpool.vercel.app/

## Release Notes

- **New Features**
  - Introduced a new font, 'Andale Mono Regular', with a corresponding CSS variable.
  - Added a Toaster component for enhanced notification capabilities.
  - Implemented multiple new metadata tags for improved social media integration.
  - Enhanced button component with a new `rounding` property for additional styling options.
  - Restructured the main page layout with new sections and interactive elements.
  - Added a new GitHub icon component for improved branding.
  - Introduced a new route for wallet connection with a dedicated button.

- **Bug Fixes**
  - Updated favicon links for better compatibility across devices.

- **Chores**
  - Added a web manifest for the application.
  - Updated dependencies to include `svelte-sonner` for toast notifications.

### CODE REVIEW

**Comment by angelicawill** [comment]
@teodorus-nathaniel please resolve conflicts, thanks!

**Comment by zolotokrylin** [comment]
Guys, please remove auth from preview links. It can be done in project's settings in Vercel.

**Comment by teodorus-nathaniel** [comment]
@zolotokrylin removed it

**Comment by teodorus-nathaniel** [comment]
@angelicawill fixed conflict

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,7 +1,94 @@
...
+				rel="noopener noreferrer"
+				size="auto"
+				rounding="full"
+				class="twsv-flex-1 twsv-gap-1.5 twsv-px-5 twsv-py-3"
+			>
+				<GithubIcon class="twsv-text-xl" />
+				<span>View on Github</span>
+			</Button>
+			<Button
+				href="https://thirdweb.com"
```
Comment (angelicawill) [inline src/routes/+page.svelte:82]: can we link this to https://portal.thirdweb.com/connect?

**Review by angelicawill — COMMENTED** [review]
Left a comment in vercel preview

**Review by angelicawill — COMMENTED** [review]
Left another comment in vercel preview, for styling

**Comment by teodorus-nathaniel** [comment]
> other comment in vercel preview, for styling

Updated

**Review by angelicawill — APPROVED** [review]

**Pull request merged**